api-specs/Gateway-EE/3.9/kong-ee.yaml

```diff
@@ -16158,6 +16230,60 @@ paths:
       tags:
       - cache
 
+  '/config':
+    post:
+      summary: Apply Declarative Config
+      tags:
+        - config
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+      requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+            application/yaml:
+              schema:
+                type: object
+            multipart/form-data:
+              schema:
+                type: object
+                properties:
+                  config:
+                    type: string
+                    format: binary
+                    example: /path/to/<config-file>
+                    description: Configuration file in JSON or YAML.
+      operationId: post-config
+      description: |
+        Apply a configuration from a declarative JSON or YAML file. Any existing configuration will be overwritten/
+
+        > Note: This API is only available in DB-less mode.
+    get:
+      summary: Get Declarative Config
+      operationId: get-config
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  config:
+                    type: string
```

Comment:
What does this return? Right now it looks like it returns "string", is it your entire config as a string?

Reply:
Yes, that's it 
![image](https://github.com/user-attachments/assets/68b3c974-ff1f-4883-8079-36866bf823f0)